cli/testdata/coder_server_--help.golden

```diff
@@ -193,6 +193,10 @@ backed by Tailscale and WireGuard.
           The maximum lifetime duration users can specify when creating an API
           token.
 
+      --proxy-health-interval duration, $CODER_PROXY_HEALTH_INTERVAL (default: 1m0s)
```

Comment:
Should this be `health-check`?

Reply:
Felt long to say `health check` and figured `health-interval` was descriptive enough